assignment.js

```diff
@@ -57,13 +65,22 @@ Use the following test cases to confirm your program meets the success criteria
 2. Given the following:
     problem1Numbers = [9, 18, 62, 3, 17]
 
-   Result: 0
+   Result: 21.8
 
 ************/
 
 console.log('Problem 2:')
 
 // Add your code below this line
+let problemNumbers = [9, 18, 62, 3, 17]
+
+let total = 0;
+for(let i = 0; i < problemNumbers.length; i++) {
```

Comment:
@rvaswani85 nice job, alternatives to using a `for` loop would be `.forEach()` or `reduce()` to calculate the sum total of all the numbers in the array